docs/concepts/collection.md

```diff
@@ -0,0 +1,29 @@
+# Collections
+
+Collections are groups of related content that can be viewed or managed as a unit.  Islandora-specific use cases include:
+
+- an archival fonds that needs to be grouped together, with internal hierarchy
+- various collections of artifacts, grouped for display
+- theses and dissertations, which are organized and managed separately from other objects.
+
+Islandora can:
+
+- declare items as members of other objects
+- after declaring an item "a collection", display a view of its members
+
+Management of the objects of a collection is not yet readily built in, but such actions as deleting, changing visibility or permissioning on a collection should be possible through Views Bulk Edit and other contrib modules, such as Permissions By Term.
+
+## Configuration provided by Islandora Defaults
+Islandora (Defaults) provides:
+
+- A membership relationship for members (“children”) to point to their parent collection
+- A View (template) to display the members of a collection on the collection’s page
+- Batch-editing tools that can be used on collections
```

Comment:
Does Islandora Defaults come with Views Bulk Edit, or what is meant by tools? It doesn't look like Islandora core or defaults require the module.

Reply:
? no idea.

Reply:
I think it simply means you _can_ use Views Bulk Edit... defaults doesn't include it as a dependency so it won't be enabled automatically. We don't have anything else at hand that does bulk editing (yet) although Islandora Workbench is getting there.

Reply:
I would probably replace the first bullet "A membership relationship for members (“children”) to point to their parent collection" to be "A Repository Item content type that includes the membership field provided by core Islandora". In technical speak, the islandora_core_feature provides the field storage for field_member_of and views that use it, but islandora_defaults actually creates a content type with field definitions that use it.

Also, as I mentioned above, we don't actually provide any batch editing tools in defaults. The members re-ordering aspect of the manage members tab, if you include that as an editing tool, is actually provided by islandora_core_feature.

Reply:
I was very deliberate in saying what islanodra provides vs what islandora defaults provides. If you think it's necessary i can mention defaults, but I think that  this is a perfect example where we can say what islandora provides (i.e. you should use this if you are using your own stuff because it's core islandora!!) 

if the distinction between putting the field / relationship in core but putting the content type in defaults is NOT meaningful, then maybe it should all be in defaults. But i chose to do this precisely because it's not defaults.

Reply:
I know you were being deliberate, which is why I was hoping to make the statement more clear about what comes from where (which I obviously didn't). 

The distinction between islandora and islandora_defaults in this case _is meaningful_. The islandora_core_feature (from the islandora module) provides the necessary basis, _field storage and views_, for adding a field_member_of and its associated functionality to *any* content type (which many of us early adopters have done) whereas islandora_defaults simply provides an _example content type that uses_ this functionality.